linux/hci/hci.go

```diff
@@ -128,12 +132,13 @@ func (h *HCI) Init() error {
 	h.evth[evt.CommandStatusCode] = h.handleCommandStatus
 	h.evth[evt.DisconnectionCompleteCode] = h.handleDisconnectionComplete
 	h.evth[evt.NumberOfCompletedPacketsCode] = h.handleNumberOfCompletedPackets
+	h.evth[evt.EncryptionChangeCode] = h.handleEncryptionChange
 
 	h.subh[evt.LEAdvertisingReportSubCode] = h.handleLEAdvertisingReport
 	h.subh[evt.LEConnectionCompleteSubCode] = h.handleLEConnectionComplete
 	h.subh[evt.LEConnectionUpdateCompleteSubCode] = h.handleLEConnectionUpdateComplete
 	h.subh[evt.LELongTermKeyRequestSubCode] = h.handleLELongTermKeyRequest
-	// evt.EncryptionChangeCode:                     todo),
+	h.subh[evt.EncryptionChangeCode] = h.handleUnhandled
```

Comment:
change to: `h.handleEncryptionChange(...)`

Reply:
fixed